index.js

```diff
@@ -15,7 +15,7 @@ function markdownitLinkifyImages (md, config) {
     var imgClass = generateClass(config.imgClass)
 
     if (token.attrIndex('title') !== -1) {
-      title = ' title="' + token.attrs[token.attrIndex('title')][1] + '"'
+      title = ' title="' + md.utils.escapeHtml(token.attrs[token.attrIndex('title')][1]) + '"'
```

Comment:
While writing the test, I found another bug in `title` attribute.
So thanks for the tip to write the test 👍